docker-compose.yml

```diff
@@ -8,9 +8,10 @@ services:
     links:
       - postgres:postgres
     volumes:
-      - /usr/src/app/static
+      - /usr/src/app/frontend/build
       - migrations:/migrations
       # ADD OTHERS VOLUMES HERE
+      - /Users/pouya/projects/neuro/shared_data:/shared_data
```

Comment:
This should probably not be commited. You can keep a local copy that doesn't get pushed with this info